prisma/schema.prisma

```diff
@@ -298,6 +299,33 @@ model CronJobs {
     @@map("cron_jobs")
 }
 
+enum AlertMetric {
+    COST_PER_USER
```

Comment:
With 'COST_PER_USER' i would expect total cost per user over all time, but this PR alerts for a single Generation. Do we want to rename this to sth. like `COST_PER_USER_PER_GENERATION`?

Reply:
It does actually check for the `total cost per user over all time`. I've added a comment to the `getTokenCostForUser` function to make this clearer, but open for your suggestions!